core/src/test/java/org/elasticsearch/index/query/FuzzyQueryBuilderTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.index.query;
+
+import org.apache.lucene.index.Term;
+import org.apache.lucene.search.FuzzyQuery;
+import org.apache.lucene.search.MultiTermQuery;
+import org.apache.lucene.search.Query;
+import org.elasticsearch.common.ParseFieldMatcher;
+import org.elasticsearch.common.lucene.BytesRefs;
+import org.elasticsearch.common.unit.Fuzziness;
+import org.elasticsearch.index.mapper.MappedFieldType;
+import org.elasticsearch.index.query.support.QueryParsers;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.Matchers.is;
+
+public class FuzzyQueryBuilderTest extends BaseQueryTestCase<FuzzyQueryBuilder> {
+
+    private static final String[] REWRITE_METHOD = new String[]{"constant_score", "scoring_boolean", "constant_score_boolean", 
+            "top_terms_1", "top_terms_boost_1", "top_terms_blended_freqs_1"};
```

Comment:
I think we have constants for this now: https://github.com/elastic/elasticsearch/blob/8238f497d85d785a61ebc017b8ac96fc5fcd28ae/core/src/main/java/org/elasticsearch/index/query/support/QueryParsers.java#L32